docs/source/get_involved/code_of_conduct.rst

```diff
@@ -133,13 +165,21 @@ project event or platform include:
 -  nonconsensual or unwelcome physical contact
 -  sustained disruption of talks, events or communications
 -  insults or put downs
--  sexist, racist, homophobic, transphobic, ableist, or exclusionary
-   jokes
+-  sexist, racist, homophobic, transphobic, ableist, or exclusionary jokes
```

Comment:
I think that part was already mentioned above under "protected characteristics"? but I dont mind to have it here again

Reply:
this is unchanged and has been in the CoC since the start. It only appears as a change because I removed a newline.